src/lib/mina_ledger/sync_ledger.ml

```diff
@@ -39,6 +39,16 @@ module Any_ledger = Syncable_ledger.Make (struct
   let account_subtree_height = 6
 end)
 
+module Root = Syncable_ledger.Make (struct
+  module Addr = Ledger.Location.Addr
+  module MT = Ledger.Root
+  module Account = Account.Stable.Latest
+  module Hash = Hash
+  module Root_hash = Root_hash
+
+  let account_subtree_height = 6
+end)
+
 module Db = Syncable_ledger.Make (struct
```

Comment:
This is now only used once in `src/lib/mina_lib/tests/tests.ml` it seems, might as well just replace all occurrences with `Sync_ledger.Root`